package.json

```
@@ -37,23 +37,26 @@
   "scripts": {
     "lerna": "lerna",
     "start": "yarn watch",
+    "clean:packages": "lerna run clean --scope=@trpc/*",
+    "clean:examples": "lerna run clean --scope=@examples/*",
+    "clean:root": "rm -rf node_modules",
+    "clean": "run-p clean:*",
```

```suggestion
    "clean:packages": "rm -rf **/node_modules **/dist",
    "clean:examples": "lerna run clean --scope=@examples/*",
    "clean": "run-p clean:*",
```
This way we won't need the `clean` scripts in each `package.json` and also won't need to have `node_modules` in each of the clean scripts for the examples.

---

Yep I believe I had something like that earlier. However, I quite like having a clean command for everyone so that when I am working on something and want to make sure nothing is "left stale", I run a `clean` for that project only. Can definitely change to your suggestion if you don't like this.